wpilibc/athena/src/IterativeRobot.cpp

```diff
@@ -161,6 +162,20 @@ void IterativeRobot::TestInit() {
 }
 
 /**
+ * Periodic code for all modes should go here.
+ *
+ * Users should override this method for code which will be called periodically
+ * at a regular rate while the robot is in any mode.
+ */
+void IterativeRobot::RobotPeriodic() {
+  static bool firstRun = true;
+  if (firstRun) {
+    std::printf("Default %s() method... Overload me!\n", __FUNCTION__);
```

Comment:
Yeah we can just make it an empty body. I did it to match all the other methods, which all have this. They only run the print once if the method is not overloaded. The projects created by the plugins do not override all the methods anyway,, so a default user program prints these anyway.